doc/user_manual/metrics.tex

```diff
@@ -599,3 +598,32 @@ \subsubsection{Distance Based Metric}
 
 In addition to this XML subnode, the users can also specify the corresponding parameters for each `metric' according to
 previous sections.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Dynamical System Scaling}
+\label{subsection:DSS}
+The Dynamical System Scaling (DSS) is a distance metrics that is used to measure the separation
+between two time-dependent data sets.
+
+The specifications of a DSS metric is defined within the \xmlNode{Metric} XML block. The XML node \xmlAttr{subType} must be \textbf{PPDSS} (see \ref{subsubsec:Validation}) in the \xmlNode{PostProcessor} for the outputs of the post-processor to be in the right format for DSS metric inputs.
+
+This XML node needs to contain the attributes:
+
+\begin{itemize}
+  \item \xmlNode{actionType}, \xmlDesc{string, required field}, the type time scaling selected for post-processor PPDSS (see \ref{subsubsec:Validation}). The options are: `DataSynthesis', `2\_2\_affine', `dilation', `beta\_strain', `omega\_strain', and `identity'.
+\end{itemize}
+
+An example of DSS  defined in RAVEN is provided below:
+\begin{lstlisting}[style=XML]
+<Simulation>
+  ...
+  <Metrics>
+    ...
+    <Metric name="example" subType="DSS">
+      <actionType>DataSynthetic</actionType>
```

Comment:
I think you have removed this node, right? It is not defined in the getInputSpecification.